04/app.js

```diff
@@ -0,0 +1,26 @@
+const divList = document.querySelectorAll('div');
+const body = document.querySelector('body');
+
+const addClass = function(){
+  
+    const self = this;
+    
+    setTimeout(function(){
+        self.classList.add('clicked');
+    }, self.dataset.time)
+  
+}
+
+const removeClass = function(e){
+    if(!e.target.className){
```

Comment:
To zabezpieczenie dlaczego?

Reply:
Dlatego, że body jako jedyny element w zadaniu nie ma klasy. Stąd chciałem, żeby kliknięcie w jedyny element bez klasy (body) usuwało klasy prostokątów :)